starport/services/scaffolder/scaffolder.go

```diff
@@ -32,6 +34,7 @@ type Scaffolder struct {
 func New(path string, options ...Option) (*Scaffolder, error) {
 	s := &Scaffolder{
 		path:    path,
+		tracer:  placeholder.New(),
```

Comment:
I think it would be better to create a new instance inside every scaffolding related funcs (e.g. CreateModule) to create isolation.

Reply:
it was this way initially, but it was pretty verbose for no good reason. what would you like to achieve with isolation?

Reply:
I mean for ex, if I create an instance of Scaffolder and run CreateModule (let's say it returns with a placeholder err) then I run AddType (but placeholders are good) from the same Scaffolder instance, AddType will still return with an error. And I think this is an unwanted situation, actually can be considered as a bug.

Reply:
agreed, that would be a bit weird